src/games/strategy/triplea/ai/proAI/ProBidAI.java

```diff
@@ -1837,10 +1841,10 @@ private static float determineEnemyBlitzStrength(final Territory blitzHere, fina
       }
     }
     for (final Unit u : unitDistance.keySet()) {
-      if (lz != null && MoveValidator.hasEnoughMovement(u, unitDistance.getInt(u) + distance.getInt(lz))) {
+      if (lz != null && Matches.UnitHasEnoughMovementForRoute(new Route(checked)).match(u)) {
```

Comment:
`new Route(checked)` call into the Matches function is redundant. Pls Overload the method to take a  `List<Territory>` as well